### ISSUE TITLE
Highlight in content with the pseudo XML used by acroReader

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#28]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?
It should be interesting to be able to get a highlight of some words in the
document, by passing a string like   <XML> <Body units=characters
color=#ff00ff mode=active version=2> <Highlight> <loc pg=0 pos=485 len=14> 
....</Body></XML>
Or it could be with a full XML compl. like
<XML> <Body units='characters' color='#ff00ff' mode='active' version='2'>
<Highlight> <loc pg='0' pos='485' len='14'/> 
....</Body></XML>
What version of the product are you using? On what operating system?

Py

Please provide any additional information below.

```

Original issue reported on code.google.com by `pierreyv...@gmail.com` on 6 Mar 2010 at 6:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@gmail.com` on 10 Mar 2010 at 7:32
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Any update on this feature request ?
```

Original comment by `Gui...@gmail.com` on 7 Dec 2010 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
I think it would be relatively easy for us to add this to our annotations 
plug-in and it really makes sense. I will make sure this gets added to the 
annotations plug-in within the next few days and update this issue once this is 
done
```

Original comment by `erik.eng...@devaldi.com` on 8 Dec 2010 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
Would it make more sense from a Javascript point of view to use a JSON object 
instead of XML ?
```

Original comment by `Gui...@gmail.com` on 10 Dec 2010 at 8:04

**Comment by GoogleCodeExporter** [comment]
```
This exact issue seems to be the limiting factor on all document viewers I can 
find online.  MY company is hoping to find a solution to displaying documents 
on the database website I am building that can take XML search results from 
another high powered search engine(DT-search) and highlight them... Much like 
acrobat Reader does(sometimes).

If your product has this functionality, I will be replacing acroReader with 
flexPaper as soon as possible.
```

Original comment by `SilvaJa...@gmail.com` on 17 Mar 2011 at 5:34

**Comment by GoogleCodeExporter** [comment]
```
Implemented in the upcoming version (1.4.2) which is due at the end of March
```

Original comment by `erik.eng...@devaldi.com` on 21 Mar 2011 at 5:27
- Changed state: **Fixed**
